graphql-server/pkg/resolvers/chain.go

```diff
@@ -0,0 +1,39 @@
+package resolvers
+
+// This file will be automatically regenerated based on the schema, any resolver implementations
+// will be copied through when generating and any unknown code will be moved to the end.
+
+import (
+	"context"
+	"time"
+
+	pkgContext "github.com/oursky/likedao/pkg/context"
+	"github.com/oursky/likedao/pkg/models"
+)
+
+func (r *queryResolver) ChainHealth(ctx context.Context) (*models.ChainHealth, error) {
+	now := time.Now()
+	currentBlock, err := pkgContext.GetQueriesFromCtx(ctx).Block.QueryLatestBlock()
+	if err != nil {
+		return nil, err
+	}
+	averageBlockTime, err := pkgContext.GetQueriesFromCtx(ctx).Chain.QueryAvergeBlockTime()
+	if err != nil {
+		return nil, err
+	}
+
+	timeSinceLastBlock := now.Sub(currentBlock.Timestamp).Seconds()
```

Comment:
i have some doubts on latency for bdjuno to index the block....

Reply:
timeSinceLastBlock is always 10+ secs actually

Reply:
how about using time between block n-1 and n-2 for congestion / healthy; & if only show halted if the last block time is way off from wall time?